### DISCUSSION TITLE
Problemas nos Contos de Fadas

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#276] [Imagens]
Post de ereferência: https://www.vish.blog.br/imagens/tirinhas/problemas-nos-contos-de-fada/

**Comment by one-eyed-willy** [comment]
A vovó deu indigestão nele

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/problemas-nos-contos-de-fada/#comment_1087533671>):

> A vovó deu indigestão nele

Ri alto aqui :D

**Reply by angellisy** [reply]
😛
